docs/intro.rst

```diff
@@ -289,6 +302,12 @@ generating
 ``target/kafka-rest-$VERSION-standalone.jar``, which includes all the
 dependencies as well.
 
+Requirements
+------------
+
+- Kafka 0.9.0.0
```

Comment:
@miguno Right, that actually changed just recently and I filed this a few days ago. Will update before merging.